editors/vscode/LICENSE

```diff
@@ -0,0 +1,31 @@
+Copyright Brent Yorgey 2021-2022
+SPDX-License-Identifier: BSD-3-Clause
+
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+
+    * Redistributions in binary form must reproduce the above
+      copyright notice, this list of conditions and the following
+      disclaimer in the documentation and/or other materials provided
+      with the distribution.
+
+    * Neither the name of Disco team nor the names of other
```

Comment:
Swarm?

Reply:
It's what's in the LICENSE file that I just copied. :man_shrugging: